library/network_connections.py

```diff
@@ -531,6 +531,17 @@ def ifcfg_create(
                 line = r["network"] + "/" + str(r["prefix"])
                 if r["gateway"]:
                     line += " via " + r["gateway"]
+                if connection["interface_name"]:
+                    line += " dev " + connection["interface_name"]
+                else:
+                    warn_fcn(
+                        "The connection '{0}' does not specify an interface name. "
+                        "Therefore, the route '{1}' will be configured without the "
```

Comment:
The result of `{1}` is very hard to read. How about
```"...the route to {1}/{2}...".format(name, r["network"], r["prefix"])```

Reply:
@ffmancera FYI

Reply:
Nice catch. I agree.

Reply:
The unit tests caught it. :grin:

Reply:
This also shows the importance of a demo as part of the review.

Reply:
Yes, you are right here. +1

Reply:
Yes, the proposed change is nice.